src/edu/washington/escience/myria/operator/Split.java

```diff
@@ -0,0 +1,124 @@
+package edu.washington.escience.myria.operator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Throwables;
+import com.google.common.collect.ImmutableMap;
+
+import edu.washington.escience.myria.DbException;
+import edu.washington.escience.myria.Schema;
+import edu.washington.escience.myria.Type;
+import edu.washington.escience.myria.column.Column;
+import edu.washington.escience.myria.column.builder.ColumnBuilder;
+import edu.washington.escience.myria.column.builder.ColumnFactory;
+import edu.washington.escience.myria.storage.TupleBatch;
+import edu.washington.escience.myria.storage.TupleBatchBuffer;
+import edu.washington.escience.myria.storage.TupleUtils;
+
+/**
+ * Operator which splits a string-valued column on a Java regular expression and duplicates the input row with each
+ * segment of the split result.
+ * 
+ * E.g., (1, 2, "foo:bar:baz") -> (1, 2, "foo"), (1, 2, "bar"), (1, 2, "baz")
+ */
+public class Split extends UnaryOperator {
```

Comment:
I do not get a checkstyle warning for this. There are other classes extending UnaryOperator which are not final (because they're extended by other classes, e.g., Apply).